src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+//Solution below slightly adapted from https://github.com/Wincenttjoi/CS2103T-duke-chatbot/blob/master/src/main/java/duke/Duke.java
+
+import duke.*;
+
 public class Duke {
+
+    private Ui ui;
+    private TaskList taskList;
+    private boolean exit;
+
+    public Duke () {
+        this.ui = new Ui();
+        this.taskList = new TaskList();
+        exit = false;
+    }
+
+    public enum InputCommands {
+        bye, list, done, delete, todo, deadline, event
```

Comment:
Since these are going to be unchanged, you could possibly rename to be capitalised? For example, BYE, LIST, DONE etc. However, i like the naming of the enum InputCommands here!